rllib/utils/metrics/stats.py

```diff
@@ -295,11 +313,23 @@ class for details on the reduction logic applied to the values list, based on
         """
         reduced, values = self._reduced_values()
 
+        # Keep track and update underlying throughput metric.
+        if self._throughput is not False:
+            # Take the delta between the new (upcoming) reduced value and the most
+            # recently reduced value (one `reduce()` call ago).
+            delta_sum = reduced - self._hist[-1]
+            assert delta_sum >= 0
+            time_now = time.perf_counter()
+            delta_time = time_now - self._throughput_last_time
+            assert delta_time >= 0.0
```

Comment:
Dumb question: Why do we need then still timing metrics? Couldn't we just log `num_env_steps_sampled_lifetime` and use a throughput?